### ISSUE TITLE
Sync client cannot login using Shibboleth on Apple OSX prior to 10.10

### ISSUE THREAD

**Issue by cdamken** [owncloud/client#2720]
We now have two occasions of users reporting that no client can login on our IdP (SimpleSAML) when using Apple OSX prior to Yosemite. When performing a fresh install of the client you can’t get past the Wizard because of this. When users select other IdP’s it works without any issue.

At the IdP side we do not see any attempt of the user to login while no Firewall is blocking the requests. When the users upgrade their Mac to the latest release (Yosemite) than the client starts to work.

In the logging we see this:

```
1-20 16:29:46:259 0x7f8323614500 networkjobs.cpp:214 !!! Mirall::CheckServerJob created for QUrl( "https://ServerPATH" ) querying "status.php"
11-20 16:29:46:314 0x7f8323614500 networkjobs.cpp:460 status.php returns: QMap(("edition", QVariant(QString, "enterprise") ) ( "installed" , QVariant(QString, "true") ) ( "version" , QVariant(QString, "6.0.4.1") ) ( "versionstring" , QVariant(QString, "6.0.4") ) ) 0 Reply: QNetworkReplyHttpImpl(0x7f83246d1730)
11-20 16:29:46:315 0x7f8323614500 connectionvalidator.cpp:99 ** Application: ownCloud found: QUrl( "https://ServerPATH/status.php" ) with version "6.0.4" ( "6.0.4.1" )
11-20 16:29:47:725 0x7f8323614500 folderman.cpp:518 FolderMan: Syncing is disabled, no scheduling.
11-20 16:29:49:224 0x7f8323614500 sparkleupdater_mac.mm:107 virtual void Mirall::SparkleUpdater::backgroundCheckForUpdate() launching background check
11-20 16:29:49:234 0x7f8323614500 sparkleupdater_mac.mm:34 -[DelegateObject updaterMayCheckForUpdates:] may check: YES
11-20 16:29:49:654 0x7f8323614500 shibbolethwebview.cpp:112 void Mirall::ShibbolethWebView::slotLoadFinished(bool) Could not load Shibboleth login page to log you in.
11-20 16:29:49:852 0x7f8323614500 sparkleupdater_mac.mm:47 -[DelegateObject updaterDidNotFindUpdate:]
11-20 16:30:15:723 0x7f8323614500 folder.cpp:260 * Polling "SURFdrive" for changes. (time since last sync: 29 s)
11-20 16:30:15:725 0x7f8323614500 folder.cpp:271 Not syncing. : false 0
11-20 16:30:18:728 0x7f8323614500 networkjobs.cpp:56 Pre-oc7 server detected, adjusting timeout values
11-20 16:30:18:729 0x7f8323614500 networkjobs.cpp:214 !!! Mirall::CheckServerJob created for QUrl( "https://ServerPATH" ) querying "status.php"
11-20 16:30:18:743 0x7f8323614500 networkjobs.cpp:460 status.php returns: QMap(("edition", QVariant(QString, "enterprise") ) ( "installed" , QVariant(QString, "true") ) ( "version" , QVariant(QString, "6.0.4.1") ) ( "versionstring" , QVariant(QString, "6.0.4") ) ) 0 Reply: QNetworkReplyHttpImpl(0x7f832660da10)
11-20 16:30:18:744 0x7f8323614500 connectionvalidator.cpp:99 ** Application: ownCloud found: QUrl( "https://ServerPATH/status.php" ) with version "6.0.4" ( "6.0.4.1" )
```

@MorrisJobke 

00002150

**Comment by MorrisJobke** [comment]
@danimo @dragotin Who of you has more experience with Mac OS X? Can you check this out? Thanks

**Comment by danimo** [comment]
@MorrisJobke To me that looks more like the matter of the client version than the OS version. Are we certain that the same client versions are involved? (i.e. 1.6.x shipped an older webkit that was known to not load the page correctly).

**Comment by danimo** [comment]
Some background, this is the central error:
> 11-20 16:29:49:654 0x7f8323614500 shibbolethwebview.cpp:112 void Mirall::ShibbolethWebView::slotLoadFinished(bool) Could not load Shibboleth login page to log you in.

It is issued by QtWebKit

**Comment by cdamken** [comment]
@danimo We have reports that 1.5.3 does work but higher not.

We have tested the situation with a local proxy (www.charlesproxy.com) in between the sync client and the server, but then the authentication does work.

So it might be something which is related to SSL , but we cannot see why because the logs do not mention anything.

**Comment by cdamken** [comment]
@danimo Any news?

**Comment by danimo** [comment]
@cdamken No, everthing works fine here. I need the _exact_ URL where this is happening.

**Comment by cdamken** [comment]
@danimo It was fixed with 1.7.1 

#### Thank you!
